oct2py/io.py

```diff
@@ -17,6 +18,9 @@
 from .utils import Oct2PyError
 
 
+_WRITE_LOCK = threading.lock()

```

Comment:
I think this needs to be `threading.Lock()`:  https://docs.python.org/2/library/threading.html#threading.Lock